entity-management-solr/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>entity-management</artifactId>
+    <groupId>eu.europeana.api</groupId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <groupId>eu.europeana.api</groupId>
+  <artifactId>entity-management-solr</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+
+  <name>entity-management-solr</name>
+  <description>A simple entity-management-solr.</description>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
```

Comment:
these properties are inherited from the base pom, please remove

Reply:
Improved.